index.js

```diff
@@ -1,22 +1,30 @@
-// tooling
-const postcss = require('postcss');
-const parser = require('postcss-value-parser');
+const postcss = require("postcss");
+const {parse} = require('postcss-values-parser');
```

Comment:
Let's also add spaces around here so it's consistent with the rest of the plugins.

```suggestion
const { parse } = require('postcss-values-parser');
```

Reply:
applied